.github/workflows/android-feature.yml

```diff
@@ -45,10 +45,3 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           arguments: jacocoTestPlayDebugUnitTestReport --stacktrace --info
-      - name: Publish test results
-        uses: dorny/test-reporter@v1
-        if: always()
-        with:
-          name: test-results
-          path: 'app/build/test-results/testPlayDebugUnitTest/TEST-*.xml'
-          reporter: java-junit
```

Comment:
why? 🤔

Reply:
As I read from https://github.com/dorny/test-reporter#recommended-setup-for-public-repositories some refactoring to the workflow is required for test results even from pull requests from forks to deliver properly without causing errors.

Shall discuss with teammates about this later.